web-api/openapi/back_office.yaml

```
@@ -446,7 +446,17 @@ components:
             Setting neither will default to `is_candidate = true`.
           type: boolean
         summarize:
-          description: Summarize the document snippets before further processing.
+          description: |-
+            Summarize the document snippets before further processing.
```

❓ shouldn't skip `snippets` here? the option refers to a single document and how we summarize it is internal details.
```suggestion
            Summarize the document before further processing.
```